app/views/vendors/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<% content_for :title do %>
+      <%= @vendor.username %>'s shop
+<% end %>
+
+<% content_for :headline do %>
+     <h1> <%= @vendor.username %>'s shop </h1>
+
+         <section>
+           <% if @vendor.name && @vendor.name != "" %>
+             <h2><%= @vendor.name %></h2>
+           <% end %>
+           <% if @vendor.description && @vendor.description != "" %>
+             <h5><strong>About Me: </strong><%= @vendor.description %></h5>
+           <% end %>
+           <% if current_vendor && current_vendor.id == @vendor.id %>
+             <p><%= link_to "Edit Profile", edit_vendor_path(@vendor.id), class: 'button small' %></p>
+           <% end %>
+         </section>
+         <%= image_tag @vendor.photo_url, alt: @vendor.username %>
+
+<% end %>
+
+
+     <% if @vendor.products.length > 0 %>
+          <h4><%= @vendor.username %>'s products</h4>
+     <% else %>
+          <h4>This user currently has no products.</h4>
+     <% end %>
+
+
+<% @vendor.products.each do |product| %>
+     <section class='product'>
+     <h4 class="name"><%= product.name %></h4>
+     <h4 class="price">$<%= sprintf('%.2f', product.price) %></h4>
```

Comment:
Might be good to create a view helper for this in the future so that you can format all prices the same way across all views without needing to duplicate this logic